vtree/diff.js

```diff
@@ -343,16 +348,23 @@ function reorder(aChildren, bChildren) {
 // TODO: We really want to make this more efficient
 // Finding the current index of any key in the array could be
 // faster.
-function moveItem(array, toIndex, itemKey) {
+function indexOfKey(array, itemKey) {
```

Comment:
> keeping it updated incrementally

the incremental updates are O(N)... still the same.  i've given this a lot of thought this week :smile: